doc/manuals/devel/httpHeaders.md

```diff
@@ -0,0 +1,329 @@
+Source IP of the HTTP request associated to the transaction, except if the request includes X-Forwarded-For header which overrides the former IP.
+X-Real-IP and X-Forwarded-For (used by a potential proxy on top of Orion) overrides IP.
+X-Real-IP takes preference over X-Forwarded-For, if both appear.
+	
+    X-Forwarded-For: 129.78.138.66
+	
+[Top](#top) 
+	
+## 10. X-Real-IP
+
+Source IP of the HTTP request associated to the transaction, except if the request includes X-Real-IP which overrides X-Forwarded-For and source IP.
+X-Real-IP and X-Forwarded-For (used by a potential proxy on top of Orion) overrides IP.
+X-Real-IP takes preference over X-Forwarded-For, if both appear.
+	
+[Top](#top)
+
+## 11.  Access-Control-Allow-Origin
+
+It is an optional header used in Orion responses.  
+Its operation is related with CORS, see specific documentation at [../user/cors.md](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/user/cors.md#access-control-allow-origin).
+    
+    Access-Control-Allow-Origin: *
+		
+[Top](#top)
+
+
+## 12. Access-Control-Allow-Headers
+
+It is a response HTTP header in Orion.
+Its operation is related with CORS, see specific documentation at [../user/cors.md](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/user/cors.md#access-control-allow-headers).
+
+    Access-Control-Allow-Headers: Content-Type, Fiware-Service, Fiware-Servicepath, Ngsiv2-AttrsFormat, Fiware-Correlator, X-Forwarded-For, X-Real-IP, X-Auth-Token	
+									               								   
+[Top](#top)		
+		
+		
+## 13. Access-Control-Allow-Methods
+
+It is a response HTTP header in Orion.
+Its operation is related with CORS, see specific documentation at [../user/cors.md](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/user/cors.md#access-control-allow-methods).
+
+[Top](#top)
+
+									  
+## 14. Access-Control-Max-Age
+
+It is a response HTTP header in Orion.
+Its operation is related with CORS, see specific documentation at [../user/cors.md](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/user/cors.md#access-control-max-age).
+		
+    Access-Control-Max-Age: 86400
+
+[Top](#top)
+
+
+## 15. Access-Control-Expose-Headers
+
+It is a response HTTP header in Orion.
+Its operation is related with CORS, see specific documentation at [../user/cors.md](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/user/cors.md#access-control-expose-headers).
+	
+    Access-Control-Expose-Headers: Fiware-Correlator, Fiware-Total-Count, Location.
+
+[Top](#top)
+
+
+## 16. Allow
+
+Allow HTTP header is used in response notification.
+It notify that which methods are allowed by the specified host/resource.
+
+    Allow: GET
+
+[Top](#top)
+		
+    
+## 17. Location
+
+Location HTTP header is used in responses.
+Response contains a Location header which holds the subscription ID: a 24 digit hexadecimal number used for updating and cancelling the subscription.
```

Comment:
Not only for subscriptions: entity creation and registration creation also use Location in a similar way. Please, upgrade the description to cover also these cases.

Reply:
Fixed in a427bd9